packages/11ty/_layouts/base.11ty.js

```diff
@@ -10,7 +10,7 @@ const { html } = require('~lib/common-tags')
 module.exports = async function(data) {
   const { pageClasses, collections, content, pageData, publication } = data
   const { inputPath, outputPath, url } = pageData || {}
-  const pageId = this.slugify(url) || path.parse(inputPath).name
+  const pageId = `page_${this.slugify(url)}` || `page_${path.parse(inputPath).name}`
```

Comment:
This statement will always return the first option, if `this.slugify(url)` is undefined the `pageId` will be `page_` rather than `page_${path.parse(inputPath).name}`. Instead you can do something like,
```suggestion
  const pageId = `page_${this.slugify(url) || path.parse(inputPath).name}`
```
or 

```
const id = this.slugify(url) || path.parse(inputPath).name
const pageId = `page_${id}`
```

Reply:
Also, we tend to use slugify with ids, so kebab-case (`page-*`) is preferable here and in the transform to snake case.